docs/release-notes/3.0.0_Beta.md

```diff
@@ -59,8 +59,7 @@ This better supports long queries and aggregation pipelines while reducing trunc
 
 ## Added monitoring support for default PostgreSQL database
 
-PMM now collects and displays metrics from the PMM's internal PostgreSQL database. While using the default database is not a recommended practice, this update ensures more complete monitoring coverage for users who rely on it.
-The change enables metrics collection from the default database across all PostgreSQL-related dashboards and QAN providing more comprehensive performance insights.
+PMM now collects metrics from its internal PostgreSQL database, displaying them in all PostgreSQL dashboards and QAN. While using the default database isn't recommended, this provides better monitoring coverage for users who rely on this setup.
```

Comment:
@catalinaadam displaying metrics from Postgres database on Postgres dashboards will be implemented in GA, so we can remove mentioning dashboards here